model/enduser/registry.yaml

```diff
@@ -0,0 +1,32 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
+    attributes:
+      - id: enduser.id
+        type: string
+        stability: experimental
+        brief: >
+          Identifier of an end user who interacts with a system.
+          This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system.
+        examples: ['QdH5CAWJgqVT4rOr0qtumf']
+      - id: enduser.role
```

Comment:
is one role enough or should it be an array? Also, do you need this field right away? If not, maybe we can keep it deprecated for now until the use-case comes?

Reply:
I can keep it as deprecated for now.